src/main/java/hudson/plugins/git/GitChangeSet.java

```diff
@@ -400,6 +401,19 @@ public User findOrCreateUser(String csAuthor, String csAuthorEmail, boolean crea
         return user;
     }
 
+    // TODO 1.651.2+ replace by API method
+    @Nullable
+    private static User getById(String id, boolean create) {
+        try {
+            return (User) User.class.getMethod("getById", String.class, boolean.class).invoke(null, id, create);
+        } catch (NoSuchMethodException x) {
+            // fine, 1.651.1 or earlier
+        } catch (Exception x) {
+            LOGGER.log(Level.WARNING, null, x);
```

Comment:
If you have nothing specific to add to the log message beyond the fact that, hey, this exception was thrown and I did not expect it, passing null is fine. [example](https://github.com/jenkinsci/support-core-plugin/blob/06293e967b8b303c66805c4b57c944c4f0d0a8b4/src/main/java/com/cloudbees/jenkins/support/SupportLogFormatter.java#L98-L100)